CONFIG.md

```diff
@@ -374,12 +373,11 @@ This is an example of the settings for the validate tasks:
   }
   "phplint": {
     "dir": [
-      "<%= config.srcPaths.drupal %>/**/*.php",
```

Comment:
(Above comment to PHPCS).

PHP-lint, on the other hand, can prevent fatal errors, and would be nice to retain for themes. Can we not assume PHP in a template.php and templates directory?